api_tests/lib/bitcoind_instance.ts

```diff
@@ -78,4 +64,32 @@ export class BitcoindInstance implements LedgerInstance {
     public getUsernamePassword() {
         return { username: this.username, password: this.password };
     }
+
+    public getDataDir() {
+        return this.dbDir.name;
+    }
+
+    private writeLogFile() {
```

Comment:
That should be something like `createConfigFile()` right?

Reply:
Yes, good catch.  Thanks.